app/conf/assets.conf

```diff
@@ -125,7 +125,7 @@ packages = {
 	openlayers = {
 		maps = OpenLayers.js,
 		mobile = OpenLayers.mobile.js,
-		stamen = http://maps.stamen.com/js/tile.stamen.js?v1.1.3
+		stamen = https://stamen-maps.a.ssl.fastly.net/js/tile.stamen.js
```

Comment:
I don't like how it's lost the version, but that's what they've got on their site.